### PR TITLE
rpc: Introduce `quorum verify`

### PR DESCRIPTION
**Pull request by UdjinM6**
Example: verifying islock sig for tx 8b5174d0e95b5642ebec23c3fe8f0bbf8f6993502f4210322871bba0e818ff3b on testnet
```
> dash-cli quorum verify 1 2ceeaa7ff20de327ef65b14de692199d15b67b9458d0ded7d68735cce98dd039 8b5174d0e95b5642ebec23c3fe8f0bbf8f6993502f4210322871bba0e818ff3b 99cf2a0deb08286a2d1ffdd2564b35522fd748c8802e561abed330dea20df5cb5a5dffeddbe627ea32cb36de13d5b4a516fdfaebae9886b2f7969a5d112416cf8d1983ebcbf1463a64f7522505627e08b9c76c036616fbb1649271a2773a1653 000000583a348d1a0a5f753ef98e6a69f9bcd9b27919f10eb1a1c3edb6c79182
true
```

### CODE REVIEW

**Review by xdustinface — CHANGES_REQUESTED** [review]
See [suggestions-3904](https://github.com/xdustinface/dash/tree/suggestions-3904) and pick what you like :)

**Review by xdustinface — DISMISSED** [review]
utACK

File: src/rpc/rpcquorums.cpp [hunk 1]
```diff
@@ -375,6 +396,39 @@ UniValue quorum_sigs_cmd(const JSONRPCRequest& request)
...
+        if (request.params[5].isNull() || (request.params[5].get_str().empty() && !request.params[6].isNull())) {
+            int signHeight{-1};
+            if (!request.params[6].isNull()) {
+                signHeight = ParseInt32V(request.params[6], "signHeight");
+            }
+            // First check against the current active set
+            quorum = llmq::quorumSigningManager->SelectQuorumForSigning(llmqType, id, signHeight, 0);
+            if (!quorum) {
+                // Then check against the previous active set in case it changed recently
+                auto signOffset = Params().GetConsensus().llmqs.at(llmqType).dkgInterval;
```
Comment (PastaPastaPasta) [inline src/rpc/rpcquorums.cpp:415]: I don't love using auto and making the type unclear. Why not just say int?

**Review by PastaPastaPasta — CHANGES_REQUESTED** [review]

File: src/rpc/rpcquorums.cpp (hunk 1 above)
Comment (UdjinM6) [reply src/rpc/rpcquorums.cpp]: done in ee2944a

**Review by xdustinface — APPROVED** [review]
utACK

**Review by PastaPastaPasta — APPROVED** [review]
utACK

**Pull request merged**